src/org/usfirst/frc/team2399/robot/OI.java

```diff
@@ -62,4 +67,35 @@ public static double getRightY() {
 	// Start the command when the button is released and let it run the command
 	// until it is finished as determined by it's isFinished method.
 	// button.whenReleased(new ExampleCommand());
+	
+	/**
+	 * Buttons
+	 */
+	
+	private static Button shootButt = new JoystickButton(leftJoy, 17);
+	private static Button stopShootButt = new JoystickButton(leftJoy, 18);
+
+	/**
+	 * Presets
+	 */
+	private static JoyShoot shooterOn = new JoyShoot(RobotMap.SHOOTER_SPEED_CONSTANT);
+	private static ShooterStop shooterStop = new ShooterStop();
+
+	public OI(){
+		shootButt.whileHeld(shooterOn);
+		stopShootButt.whenPressed(shooterStop);
```

Comment:
How would this control be used if the default behavior is to stop shooting when the button is released?